app/src/main/java/waiter/Transporter.java

```diff
@@ -0,0 +1,8 @@
+package waiter;
+
+
+import java.io.IOException;
+
+public interface Transporter {
+    void transport(Connection client) throws IOException;
```

Comment:
Same as the `Awaiter` interface. Do we need this `Transporter` interface?

Reply:
I included this interface so I can easily mock the Messenger without having to override the Messenger class's functions

Reply:
hmm. So this interface is to make sure that the mock and the actual class have the same interface(void transport method)?